clmm/data__types.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
This file name has two underscores? Is this intended?

Reply:
It also has the same name as clmm/core/datatypes.py

Reply:
Actually, was the file just moved? What happened to the one in clmm/core?

Reply:
@matthewkirby as far as I could find out, the file clmm/core/datatypes.py is the one being imported by all other modules. I dont know why clmm/data__types.py exists.
@combet do you know what is this purpose of this file? Is it actually being used? I could not find an import of it in any other part.

Reply:
@matthewkirby  -  I think all requested changes were made.  It looks like you have to approve the requested changes(?)